app/models/reports/result.rb

```diff
@@ -180,9 +180,10 @@ def calculate_period_info(range)
         hsh[period] = {
           name: period.to_s,
           start_date: period.start_date.to_s(:day_mon_year),
-          ltfu_since_date: period.start_date.advance(months: -12).to_s(:day_mon_year),
+          ltfu_since_date: period.end_date.advance(months: -12).to_s(:day_mon_year),
```

Comment:
This is an unrelated fix, right? Nice catch.

Reply:
This should be the `start_date`, no? The LTFU cards show data as of the beginning of the current month (`1-Feb-2021` instead `28-Feb-2021`).

Reply:
Should this be `period.bp_control_end_date.advance...`? The LTFU patients we shown are as of `1 Feb 2021` and not `28 Feb 2021`. We want the end of the previous month here, i.e `31 Jan 2021` I think.


Let's also rename this to something like `ltfu_bp_not_taken_since_date` since this name suggests these were lost to follow up an year ago, which means they didn't have a BP since 2 years.😅

Reply:
@prabhanshuguptagit can you help confirm which date the query's using? Once we know, we can make sure:
* It's the intended date
* It's consistent with the copy

Reply:
The query returns patients ltfu as of `period.start_date`. So if we are looking at the chart in february (say Feb 15th):
- Patients breakdown is as of Feb 1
- Last data point on LTFU chart is as of Feb 1

Reply:
Thanks for the clarification @prabhanshuguptagit .

@claudiovallejo , that means the change you've made above makes the copy inconsistent with the query. Let's:
* Revert this change for now, to ensure the copy is accurate
* Find out what the desired LTFU behavior is (if we haven't already)
  * Should the Jan 2021 report show an LTFU breakdown as of Jan 1, or Jan 31?
  * If we're looking at the current month (Feb 2021), is it ok for our LTFU query to run from Feb 29, 2020 - Feb 26 (today) 2021?
* Update the copy and query together if needed.

Reply:
Thanks @harimohanraj89 - I reverted the changes and will discuss desired LTFU behavior with Reena and Daniel :)